services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/core/validation.py

```diff
@@ -260,19 +262,16 @@ async def validate_compose_spec(
     )
 
     # validate against docker-compose config
-
-    command = "docker-compose --file {file_path} config"
-    finished_without_errors, stdout = await write_file_and_run_command(
+    finished_without_errors, stdout = await docker_compose_config(
+        compose_spec,
         settings=settings,
-        file_content=compose_spec,
-        command=command,
-        command_timeout=None,
+        command_timeout=docker_compose_config_timeout,
     )
+
     if not finished_without_errors:
-        message = (
-            f"'docker-compose config' failed for:\n{compose_spec}\nSTDOUT\n{stdout}"
+        logger.warning(
```

Comment:
if I understand right you start with a short timeout, so that we can see in the logs?
Cause I guess we cannot be sure that the underlying host is always in the best shape and therefore sometimes it can be slow. Shall we then report an error if it goes above some threshold, so that someone may re-schedule the sidecar somewhere else? or just wait longer?

Reply:
@sanderegg  Please check how I added timeout options now in the request queries

Let's look at this offline.

Reply:
checked with @sanderegg